Adding Tls client information (#1344)
temporalio/documentation: docs/typescript/how-to-set-the-namespace-for-a-temporal-client-in-typescript.md

id: how-to-set-the-namespace-for-a-temporal-client-in-typescript
title: How to set a Namespace for a Temporal Client in TypeScript
sidebar_label: Set Namespace

> description: Set the Namespace method on Workflow Client Options Builder.

Reviewer: ```suggestion
description: Set the Namespace in Workflow Client Options Builder.
```
Does the sentence remain technical correct after my edit?
